### PR TITLE
A+C goes phygital

### PR DESCRIPTION
**Pull request by elRadish**
New blog post from cologne.

### CODE REVIEW

**Comment by elRadish** [comment]
Image file links are still absolute for reviewing - shall I fix that?

File: _posts/2016-7-26-Phygital.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+Building a small wireless sensor for a prototyped "Phygital magazine"
+
+
+## Preface
+
+The goal of the described project is to build a small short-distance wireless sender equipped with a touch or light sensor that can be hidden in a printed magazine / book to make it "Phygital" = Physical and Digital. 
+Two use-cases have been implemented: 
+
+* The user sees a printed advertisment with an integrated touch area. If he touches the area in the paper, the corresponding product is offered on a nearby screen and can be customized by the reader.
+* The user opens a book and the reading light switches oh while the ambient light dims down.
```
Comment (fabue) [inline _posts/2016-7-26-Phygital.md:14]: typo "on"

File: _posts/2016-7-26-Phygital.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+## Requirements
+* Small
```
Comment (fabue) [inline _posts/2016-7-26-Phygital.md]: Small size

File: _posts/2016-7-26-Phygital.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+* As simple as possible
+* Maintenance free
+* Power efficient 
+* Fast reaction on user action
+* No LiPo batteries because of fire hazard while charging them (especially between paper)
+
+## Wireless technology
+The follwoing criteria were taken into account when choosing the wireless technology:
+
```
Comment (fabue) [inline _posts/2016-7-26-Phygital.md:26]: No sentence delimiter in lists

File: _posts/2016-7-26-Phygital.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+We've chosen a very simple 433Mhz [ASK-Modulated](https://en.wikipedia.org/wiki/Amplitude-shift_keying) technology, often called "RF433". They are very similar to the 433Mhz wireless modules used for wireless power outlets, for cheap headphones, simple doorbells etc.
+They work _nearly_ like a wire - they just transmit the level on the transmitter's input pin to the receiver's output pin. In an ideal world, we could just use them as wire and connect them to the UART-pins of the microcontroller. That even works sometimes, but is not reliable enough. It needs a protocol that deals better with the specific characteristics of wireless transmission.
+
+![image](https://github.com/elRadish/artcom.github.io/blob/phygital/images/2016-7-26-Phygital/rf433.jpg)
+
+*RF 433 sender and receiver*
+
+The whole transmitting logic is simple and can be done in software. It has to deal with bad receiption and deals with things like sending a preamble to give the receiver a chance to adjust its auto gain control to the signal. Additionally, the data is encoded 4-to-6 bits to achieve a better [DC-bias](https://en.wikipedia.org/wiki/DC_bias) for wireless transmission.
```
Comment (fabue) [inline _posts/2016-7-26-Phygital.md]: typo "receiption"

File: _posts/2016-7-26-Phygital.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+
+There are a couple of useful libraries for this purpose. [VirtualWire](https://www.pjrc.com/teensy/td_libs_VirtualWire.html), [ManchesterTX](http://mchr3k.github.io/arduino-libs-manchester/), [RadioHead](http://www.airspayce.com/mikem/arduino/RadioHead/). We've chosen VirtualWire, since it has a small footprint and is well documented. As a nice feature, it handles transimtting via interrupt service routines in the "background" (If we can talk about "background" on a 8 Mhz single core microcontroller ;-) ).
```
Comment (fabue) [inline _posts/2016-7-26-Phygital.md]: typo "transimtting"

File: _posts/2016-7-26-Phygital.md [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+* Green: Connected to MQTT-Broker
+* Blue: Receiving wireless message
+* Flashing pink: Transmitter battery low
+
+![image](https://github.com/elRadish/artcom.github.io/blob/phygital/images/2016-7-26-Phygital/receiver.png)
+
+*Receiver*
+
+## Touch
+To keep it as simple as possible, I preferred the simplest touch principle - resistive touch between two (nearly) invisible contacts. They are painted with black conductive color onto a black area of the magazine page. Contact is made with self adhesive copper tape on the back of the page.
```
Comment (fabue) [inline _posts/2016-7-26-Phygital.md]: Either "I" or "We"

**Pull request merged**